queries.sql

```diff
@@ -1,3 +1,80 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals
+WHERE name LIKE '%mon';
+
+SELECT name FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals
+WHERE neutered=true AND escape_attempts<3;
+
+SELECT date_of_birth FROM animals
+WHERE name='Agumon' OR name='Pikachu';
+
+SELECT name, escape_attempts FROM animals 
+WHERE weight_kg>10.5;
+
+SELECT * FROM animals 
+WHERE neutered=true;
+
+SELECT * FROM animals 
+WHERE name <> 'Gabumon';
+
+SELECT * FROM animals 
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+SAVEPOINT SP4;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP5;
+
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+
+UPDATE animals
+SET weight_kg = weight_kg * (-1)
+WHERE weight_kg < 0;
+
```

Comment:
You have skipped something that is required here, after updating and setting the weight, you don't the where clause instead, you are to revert back to the previous save point and then perform an update again as shown below. Consider making this adjustment in order to see what is actually happening in the DB.

Like this:

```
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```